tests/backup/backup_locked_bucket_test.go

```diff
@@ -406,3 +406,233 @@ var _ = Describe("{LockedBucketResizeOnRestoredVolume}", func() {
+						log.InfoD(fmt.Sprintf("get volumes for %s app", ctx.App.Key))
+						appVolumes, err := Inst().S.GetVolumes(ctx)
+						dash.VerifyFatal(err, nil, fmt.Sprintf("Fetch volumes for app %s", ctx.App.Key))
+						dash.VerifyFatal(len(appVolumes) > 0, true, "Verifying if app volumes exist")
+						var requestedVols []*volume.Volume
+						log.InfoD(fmt.Sprintf("Increase volume size %s on app %s's volumes: %v",
+							Inst().V.String(), ctx.App.Key, appVolumes))
+						requestedVols, err = Inst().S.ResizeVolume(ctx, Inst().ConfigMap)
+						dash.VerifyFatal(err, nil, "Verifying volume resize")
+						log.InfoD(fmt.Sprintf("validate successful volume size increase on app %s's volumes: %v",
+							ctx.App.Key, appVolumes))
+						for _, volume := range requestedVols {
+							// Need to pass token before validating volume
+							params := make(map[string]string)
+							if Inst().ConfigMap != "" {
+								params["auth-token"], err = Inst().S.GetTokenFromConfigMap(Inst().ConfigMap)
+								dash.VerifyFatal(err, nil, "Fetching token from configMap")
+							}
+							err := Inst().V.ValidateUpdateVolume(volume, params)
+							dash.VerifyFatal(err, nil, fmt.Sprintf("Validate volume %v update status", volume))
+						}
+					}
+				})
+				Step("Checking size of volume after resize", func() {
+					log.InfoD("Checking size of volume after resize")
+					pods, err := core.Instance().GetPods(namespace, labelSelectors)
+					dash.VerifyFatal(err, nil, fmt.Sprintf("Fetching the pod list"))
+					srcClusterConfigPath, err := GetSourceClusterConfigPath()
+					dash.VerifyFatal(err, nil, fmt.Sprintf("Getting kubeconfig path for source cluster %v", srcClusterConfigPath))
+					for _, pod := range pods.Items {
+						afterSize, err := getSizeOfMountPoint(pod.GetName(), namespace, srcClusterConfigPath)
+						dash.VerifyFatal(err, nil, fmt.Sprintf("Fetching the mount size %v", afterSize))
+						podListAfterSizeMap[pod.Name] = afterSize
+					}
+					for _, pod := range pods.Items {
+						dash.VerifyFatal(podListAfterSizeMap[pod.Name] > podListBeforeSizeMap[pod.Name], true, fmt.Sprintf("Verifying volume size for pod %s", pod.Name))
+					}
+				})
+				Step("Create schedule policy after initializing volume resize", func() {
+					log.InfoD("Create schedule policy after initializing volume resize")
+					ctx, err := backup.GetAdminCtxFromSecret()
+					dash.VerifyFatal(err, nil, "Fetching px-central-admin ctx")
+					periodicSchedulePolicyName = fmt.Sprintf("%s-%v", "periodic", time.Now().Unix())
+					periodicSchedulePolicyNames = append(periodicSchedulePolicyNames, periodicSchedulePolicyName)
+					periodicSchedulePolicyUid = uuid.New()
+					periodicSchedulePolicyUids = append(periodicSchedulePolicyUids, periodicSchedulePolicyUid)
+					periodicSchedulePolicyInfo := Inst().Backup.CreateIntervalSchedulePolicy(5, 15, 5)
+					err = Inst().Backup.BackupSchedulePolicy(periodicSchedulePolicyName, periodicSchedulePolicyUid, orgID, periodicSchedulePolicyInfo)
```

Comment:
It is recommended to use a single schedule policy for multiple schedule backups.

Reply:
Thanks!